translations/es.ts

```diff
@@ -2349,13 +2349,13 @@ Si no estás seguro, selecciona &quot;No&quot; para evitar comunicaciones usando
     <message>
         <location filename="../src/widget/widget.cpp" line="497"/>
         <source>Toxcore failed to start, the application will terminate after you close this message.</source>
-        <translation>toxcore falló al iniciar, el programa terminará al cerrar este mensaje.</translation>
+        <translation>Toxcore falló al iniciar, el programa terminará al cerrar este mensaje.</translation>
```

Comment:
I took irungentoo's statistics from his indiegogo campaign as reference. [Here](https://www.indiegogo.com/projects/toxcore-development) most of them start with lower case.